src/poll.rs

```diff
@@ -1,19 +1,30 @@
-#[cfg(any(target_os = "linux", target_os = "android"))]
+#[cfg(any(target_os = "linux", target_os = "android", target_os = "freebsd", target_os = "dragonfly"))]
```

Comment:
Can you alphabetize the list of target OSes here since you're already modifying it?